README.md

```diff
@@ -163,10 +170,12 @@ initialization scripts (and later daemons) to pick from.
 ovs-vsctl set Open_vSwitch . external_ids:k8s-api-server="$OVN_CENTRAL_IP:8080"
 ```
 
-Install the following additional packages for python.
+Clone this repository and install the executables.
```

Comment:
@salv-orlando 

I updated the docs to say just 'pip install .' in v2
Does that not mean installing executables? Because I do see following exes (ovn-k8s-cni-overlay, ovn-k8s-overlay, ovn-k8s-watcher) in /usr/local/bin/